### DISCUSSION TITLE
Are weight files available for large models?

### DISCUSSION THREAD

**Discussion by kadirnar** [VuongTuanKhanh/Funix-Capstone-Project#4] [Ideas]
Have you tried it on Yolov5-M and Yolov5-X models?

**Comment by VuongTuanKhanh** [comment]
> Have you tried it on Yolov5-M and Yolov5-X models?

Actually not. But I do recommend you not to use those two models. Speed is the most important thing that need to be consider for the detection problem of this issue. So Yolov5s - the lightest and fastest version of yolov5 should be the best option.

**Reply by kadirnar** [reply]
I didn't get the performance I wanted from this model. I want to create a YOLOV5 model that will detect uav in these videos. I guess the Airborne object dataset is not suitable for this.

Youtube Channel: https://www.youtube.com/c/MHAVKFPVTEAM
Run Code:
``` 
yolov5 detect --source https://www.youtube.com/watch?v=Aid03a-tBbI --weights .\best_weights.pt --view-img
````
My Work: https://github.com/kadirnar/combat-drone/blob/main/videos/uav.gif

**Reply by VuongTuanKhanh** [reply]
> I didn't get the performance I wanted from this model. I want to create a YOLOV5 model that will detect uav in these videos. I guess the Airborne object dataset is not suitable for this.
> 
> Youtube Channel: https://www.youtube.com/c/MHAVKFPVTEAM Run Code:
> 
> ```
> yolov5 detect --source https://www.youtube.com/watch?v=Aid03a-tBbI --weights .\best_weights.pt --view-img
> ```
> 
> My Work: https://github.com/kadirnar/combat-drone/blob/main/videos/uav.gif

What is your highest performance that you get ?

**Reply by kadirnar** [reply]
I want to use pytorch supported yolo model. That's why I need to retrain. But it has trouble detecting small objects. So I need Airborne dataset. But I don't have the equipment to train the big dataset.
